Makefile

```diff
@@ -121,14 +127,16 @@ $(INSTALL_DIR) :
 ###################### CLEAN/UNINSTALL TARGETS ################################
 ###############################################################################
 # clean
-clean : 
+clean : uninstall_python_requirements
 	-$(RM_DIR) $(CMAKE_BUILD_DIR)
 	-$(RM_DIR) $(PYTHON_BUILD_DIR)
 	-$(RM) $(SYMLINK_SERVICES_DIR)
 	-$(RM) $(SYMLINK_PLUGINS_DIR)
 
 # uninstall
```

Comment:
I don't think we should have an uninstall - I have never seen it in make. 
I would remove this whole section.